package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "react-accessible-treeview",
   "description": "A react component that implements the treeview pattern as described by the WAI-ARIA Authoring Practices.",
-  "version": "2.10.0",
+  "version": "2.11.0",
```

Comment:
https://github.com/dgreene1/react-accessible-treeview/pull/211 Should merge this bump
@mellis481